<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>874</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" rowspan="2">
    <widget class="QGroupBox" name="gb_connection">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>313</width>
       <height>320</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>9999</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-image:url(&quot;:/connection_sketch/1-4.png&quot;);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignJustify</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="gb_setting">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>661</width>
       <height>362</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>99999</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#gb_setting{
        border-image:url(:/connection_sketch/setting_pic.png)}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QComboBox" name="band_select">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>69</y>
        <width>113</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>113</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>ALL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2.4 Ghz</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5 Ghz</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cbo_total_port">
      <property name="geometry">
       <rect>
        <x>224</x>
        <y>176</y>
        <width>73</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>2 port</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4 port</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8 port</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="offset_24xx">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>265</y>
        <width>91</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="offset_5xxx">
      <property name="geometry">
       <rect>
        <x>328</x>
        <y>265</y>
        <width>91</width>
        <height>28</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_save_all_port_loss">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>228</y>
        <width>115</width>
        <height>81</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>75</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>save
result</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_measure_loss">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>119</y>
        <width>115</width>
        <height>101</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>98</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>measure
loss</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbo_port_option">
      <property name="geometry">
       <rect>
        <x>224</x>
        <y>122</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QCheckBox" name="use_localhost">
      <property name="geometry">
       <rect>
        <x>116</x>
        <y>82</y>
        <width>21</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="cbo_ip_destination">
      <property name="geometry">
       <rect>
        <x>224</x>
        <y>69</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>131</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="QTableView" name="tableView">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>682</height>
      </size>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>60</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QProgressBar" name="progressBar">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="format">
      <string>%p%</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
